### ISSUE TITLE
No way to access Quick clean page with C cleaner pro

### ISSUE THREAD

**Issue by darkempathy** [nvaccess/nvda#9245]
Hi. 

I've been using C cleaner from http://web2.web.piriform.com/ for many years now (actually longer than I've been using NVDA, and for the most part NvdA has been fine with it. 

A recently added feature however is quick clean, which provides an extra property page which can perform a quick scan to remove trackers and other junk without doing a  harddrive scan. 

If I tab round the C cleaner interface with NvdA, it reads all of the buttons until I get to the quick clean page. NVdA just announces property page then hitting tab or arrows does nothing. I can read the page using NVdA plus B, but using object navigation does not work.

This means there is no way to access the buttons on the quick  property page other than hitting tab a few times and hoping to hit the right button, it also means that if I accidentally get stuck on the property page its rather hard to get out of it since ctrl tab does not work either. 

I really do not know why NvdA is having trouble with this specific part of the program, when everything else in C cleaner is %100 fine, but if it could be fixed I'd much appreciate that, since C cleaner is a very useful program and one I (and many other people), use regularly to keep Pcs free of junk.

thanks in advance.

**Comment by Brian1Gaff** [comment]
Yes I'm pretty sure many have reported this to them, but thus far its been 
left a bit like some of the settings, not very accessible. It could be 
possible to get an app module tweaked, I do not know, but I personally feel 
that Ccleaner has now gone into the realms occupied by Avast etc, of sod the 
blind user.
 Indeed if you download it from their own web site it often secretly 
installs avast and at some future time will suddenly install it.
 I now only download the portable version and to be honest at the moment the 
version I have has a quick clean but I disabled it and now use it like the 
older version.
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
-----

**Comment by darkempathy** [comment]
While I don't disagree with you about antivirus programs, at the same time its only been the Quick clean option I've noticed in C cleaner which has been an issue with NvdA, and the page even reads correctly. 
This is why I wondered if there could be a fix in NVdA to cover this specifically, as I suspect its just a bit of element labelling that is required.

**Comment by Adriani90** [comment]
The health check is focusable now in last version of -CCleaner with NVDA 2020.1. Closing as works for me.
